Sources/Purchasing/StoreKit1/StoreKit1Wrapper.swift

```diff
@@ -262,3 +281,23 @@ extension StoreKit1Wrapper: SKPaymentQueueDelegate {
 // @unchecked because:
 // - Class is not `final` (it's mocked). This implicitly makes subclasses `Sendable` even if they're not thread-safe.
 extension StoreKit1Wrapper: @unchecked Sendable {}
+
+fileprivate extension SKPaymentTransactionState {
+
+    var diagnosticsName: String {
```

Comment:
TBH, since this will be an iOS-only property, I don't care that much to follow the uppercase naming... But might be good to be consistent 👍